### PR TITLE
Portfolio asset tree design

### PR DESCRIPTION
**Pull request by HamedRajabiVaramini**
Portfolio asset tree design

## Changes
- Implements ExpandShape component
- Moves out MatchSearch enum and TreeNodeData type 
- Implements HorizontalSeparator component
- Implements Toolbar component 
- Moves out TreeView and TreeNode 
- Applies some styles on buttons and checkboxes
- Performs tree design

## Fix Issues
- Renames TreePanel.tsx file to tree-panel.tsx

### CODE REVIEW

**Comment by jackieli-tes** [comment]
@HamedRajabiVaramini I ran this locally and looks fine. The file name of `src/components/toolbar/toolbar.tsx` should be `Toolbar.tsx`

Can you also run the formatter, your editor should have it hooked up, if not, you can run `npx prettier -w '**/*.tsx'`

File: src/components/expand-shape/style.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+@keyframes expandAnimation {
+  from {
+    transform: rotate(90deg);
+  }
+  to {
+    transform: rotate(180deg);
+  }
+}
+
+@keyframes collapseAnimation {
```
Comment (jackieli-tes) [inline src/components/expand-shape/style.css:10]: I'm not sure the collapse & expand keyframes are setup properly. Initial frame shouldn't have 180 rotation. it causes the arrow to animate on show

Comment (jackieli-tes) [reply src/components/expand-shape/style.css:10]: See this: 

https://user-images.githubusercontent.com/64778297/224664455-b89079cd-1b6d-4f0e-9cbd-668c2550049a.mov

Comment (HamedRajabiVaramini) [reply src/components/expand-shape/style.css:10]: Fixed.

**Review by jackieli-tes — APPROVED** [review]

**Pull request merged**